### PR TITLE
Fix number of Renko bricks being drawn when trend switches directions

### PR DESCRIPTION
**Pull request by coffincw**
Small fix to draw one less brick when the trend changes directions to account for the brick already drawn.

### CODE REVIEW

**Comment by coffincw** [comment]
huh that is odd, I didn't test with the notebook data this time so I didn't notice the difference.  I'll check this out now

**Comment by coffincw** [comment]
So the reason the bricks were appearing as rectangles was due to the datalim being updated which stretches the x-axis and subsequently the bricks.  This change should fix it.

**Comment by DanielGoldfarb** [comment]
> So the reason the bricks were appearing as rectangles was due to the datalim being updated which stretches the x-axis and subsequently the bricks. This change should fix it.

This is very confusing to me.   I don't think we want to skip adjusting the x,y axis limits altogether for renko.   Previously it was working, because **xdates got appropriately re-set after calling `_construct_renko_collections()`** based on the new_dates returned from `_construct_renko_collections()`.   It seems to me, just off the top of head, that more likely with the new algorithm new_dates is not the correct length??   And/or the actual dates being put into new_dates are not entirely correct?

I am very nervous to skip this block of code for any particular type, unless you can convince why it would work.

File: src/mplfinance/plotting.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def plot( data, **kwargs ):
         collections = _construct_ohlc_collections(xdates, opens, highs, lows, closes,
                                                          marketcolors=style['marketcolors'] )
     elif ptype == 'renko':
-        collections, new_dates, volumes, brick_values = _construct_renko_collections(dates, highs, lows, volumes, config['renko_params'], closes,
+        collections, new_dates, volumes, brick_values, brick_size = _construct_renko_collections(dates, highs, lows, volumes, config['renko_params'], closes,
```
Comment (DanielGoldfarb) [inline src/mplfinance/plotting.py:267]: _construct_renko_collections() in _utils.py is not returning `brick_size`.    other than that, all looks good.

Comment (coffincw) [reply src/mplfinance/plotting.py:267]: Oops, I'll fix that now

**Pull request merged**